_pages/infrastructure/monitoring.md

```diff
@@ -3,28 +3,60 @@ title: Monitoring
+* **Clear expectations of uptime & availability:** At present, 18F staff work 40 hour weeks and there is no requirement to be available in off hours. In practice, people want to make sure their stuff works, and many will jump online to fix things if they see a problem over the weekend. But **there no expectation of this**. Furthermore, **this understanding must be established with project partners.** Projects that need greater support coverage should arrange dedicated on-call staff elsewhere.
+
+
+## Errors & Performance Problems
 
 For a non-static site, you will want to know if exceptions are being thrown within your application. Options:
 
 * [New Relic APM](http://newrelic.com/application-monitoring)
-    * See [the FISMA Ready instructions](https://github.com/fisma-ready/new-relic).
+    * See [the FISMA Ready instructions](https://github.com/fisma-ready/new-relic) for useful settings to go in the New Relic `ini` file.
+    * At time of writing, [New Relic Alerts](https://docs.newrelic.com/docs/alerts/new-relic-alerts/getting-started/new-relic-alerts) is in open beta. While it doesn't yet provide all the features of the legacy system, it's already enough of an improvement that it's worth checking out.
+    * For [New Relic](https://newrelic.com) access, [open an issue in the DevOps repo](https://github.com/18F/DevOps/issues/new?title=New+Relic+account+for+%3Cname%3E) to get an account set up for your project.
 * [Sentry](https://getsentry.com/)
-    * You will need to deploy/manage this yourself—see the [deployment instructions](https://github.com/18F/how-to-deploy/blob/master/sentry/README.md).
+    * You can deploy/manage this yourself—see the [deployment instructions](https://github.com/18F/how-to-deploy/blob/master/sentry/README.md). However, @jmcarp has set up [sentry.18f.gov](https://sentry.18f.gov/) for 18F-wide use. _(Moar docs needed here!)_
+
+### Alert Conditions
+
+Error & performance monitors can trigger alerts on a number of different conditions, including:
+
+* Error counts (total or percentage)
+* [Apdex score](http://apdex.org/overview.html) (a responsiveness statistic)
+* Throughput
+* Response time
+* Custom metric (which can be sent to monitors for logging using the monitor's  client library)
 
-### Downtime
+All of the above can be set with thresholds for given time periods; for example, alerting if more than 2% of transactions in any five-minute period return errors.
+
+We recommend creating a mixture of alert conditions during development and tuning them based on the current performance of the app. You may have an Apdex target of 0.9, but if the app is regularly scoring lower then it's counter-productive to keep that as an alert threshold: you'll just fill the alerts with noise that can't be dealt with quickly. The work to meet that performance should be managed at the project level.
+
+Once you've created alert conditions, ensure that they're actually
```

Comment:
Unfinished sentence here. Also, I think we can be more opinionated in this section, giving more explicit guidance about what alerts to create at what thresholds. We can totally deal with that in a follow-up PR if you prefer, though.